### ISSUE TITLE
Search function in "Variables Tab"

### ISSUE THREAD

**Issue by McHauge** [bitfocus/companion#1899]
### Is this a feature relevant to comapnion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

Would love to have the options to "search" the list of variables from a module, just like the search option when adding a module.

### Usecases

The reason this need has come is that modules are going from less than 100 variables to some having more than 1000 variables. and if was simpler to sort and lookup stuff it would give a lot more insight into what you can do with a given module.

I know you can also "search" when using the variables, but that does not provide any feedback or overview of what to expect as you don't get the line by line thing with the current value and so on.

so basically would be nice to have this:
![image](https://user-images.githubusercontent.com/31018035/153045843-0d8d7781-67f3-4eb6-8a19-8728963d8f7f.png)

but in here:
![image](https://user-images.githubusercontent.com/31018035/153045899-3f0c79ce-af6d-4ad5-bc26-bfadb255c67e.png)

Also by the same notes maybe folders would also make sense to have, not needed, but might be nice to have, so modules can sort out their variables depending on what they reflect.

**Comment by Julusian** [comment]
This is in the next beta build
